docs/lifespan.rst

```diff
@@ -0,0 +1,76 @@
+Lifespan
+==============================
+The Lifespan QoS establish the maximum validity period of the samples saved on the entity history. Each of those
+samples has a lifespan timer associated so that once it expires the linked data is removed from the entity history.
+
+Before starting with the example, we need to advise you that it is complicated to understand.
+
+In this test, we are going to create two Publishers and two Subscribers, one using lifespan and the other
+without it, so you can see graphically the difference between both cases.
```

Comment:
I liked that you warned above. However, I would say it in a different way (you can even use the sentence above which I said was optional):

> Unlike other QoS, such as Deadline or Liveliness, this Qos does not provide a means to inform the user that the sample is being removed, which makes this test more complicated to illustrate. For this reason, we are going to create two publishers and two subscribers, and make only one of them use Lifespan, so that the effect of using this QoS can be graphically seen.